MLlib/loss_func.py

```diff
@@ -38,3 +38,15 @@ def derivative(X,Y,W):
 		AbsError=(np.dot(X,W).T-Y)
 		return np.dot(np.divide(AbsError,np.absolute(AbsError),out=np.zeros_like(AbsError), where=(np.absolute(AbsError))!=0),X).T/M
 
+class SparseCategoricalCrossEntropy():
+    @staticmethod        
+    def loss(X, Y, W):
+        M=X.shape[0]
+        H=sigmoid(np.dot(X,W).T)*(len(Y)-1)
```

Comment:
use `softmax` and use input `one hot encoder` and also use `assert` to check if the given `y` is one hot encoder.